libraries/AP_Compass/AP_Compass_config.h

```diff
@@ -112,6 +112,10 @@
 #define AP_COMPASS_MMC5XX3_ENABLED AP_COMPASS_I2C_BACKEND_DEFAULT_ENABLED
 #endif
 
+#ifndef AP_COMPASS_QMC5883P_ENABLED
+#define AP_COMPASS_QMC5883P_ENABLED AP_COMPASS_I2C_BACKEND_DEFAULT_ENABLED
```

Comment:
to reduce impact on low flash boards I think we need && BOARD_FLASH_SIZE > 1024

Reply:
We should probably get into the habit of putting these into the various minimize includes

Reply:
Done this change